builtin/mainmenu/dlg_settings_advanced.lua

```diff
@@ -832,6 +838,8 @@ local function handle_change_setting_buttons(this, fields)
 			for _, name in ipairs(setting.possible) do
 				if checkboxes["cb_" .. name] then
```

Comment:
This if needs to be wrapped in a `if name:sub(1, 2) ~= "no"` so you don't save e.g. `nonocaves`.

Reply:
Fixed.